modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/upgrade/UpgradeJSPGetImagePreviewURLMethodCheck.testjsp

```diff
@@ -0,0 +1,22 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/repository_entry_browser/init.jsp" %>
+
+	<div class="item-preview" data-href="<%= Validator.isNotNull(thumbnailSrc) ?
+	HtmlUtil.escapeHREF(DLUtil.getImagePreviewURL(fileEntry, themeDisplay))
+	 : themeDisplay.getPathThemeImages() + "/file_system/large/default.png" %>">
+	</div>
```

Comment:
This look a little strange. Specifically, the indentation looks off (the starting `<div>` should have no indentation). Is this formatted?

Reply:
Ok! I did run formatSource to better this. 
Done!